packages/editor/src/store/selectors.js

```diff
@@ -300,7 +300,7 @@ export function isCurrentPostPublished( state ) {
 	const post = getCurrentPost( state );
 
 	return [ 'publish', 'private' ].indexOf( post.status ) !== -1 ||
-		( post.status === 'future' && moment( post.date ).isBefore( moment() ) );
+		( post.status === 'future' && ! isInTheFuture( post.date, 1 ) );
```

Comment:
Will a future maintainer understand what the second argument it is by glancing at it? It confuses me just now looking at it.

Reply:
Can we prepare this ahead of time? If it's an ISO-8601 string, could we just do `new Date( post.date ) + MINUTE_IN_MS`